bucket/git-with-openssh.json

```diff
@@ -25,6 +25,11 @@
         "usr\\bin\\sftp.exe",
         "usr\\bin\\ssh-add.exe",
         "usr\\bin\\ssh-agent.exe",
+        [
+            "bin/sh.exe",
+            "ssh-copy-id",
+            "ssh-copy-id"
```

Comment:
Is there any reason to have this line duplicated?

Reply:
Yes; this is using the alias shim format per the [manifest docs](https://github.com/lukesampson/scoop/wiki/App-Manifests#:~:text=bin%3A), which is of the form `[ "program.exe", "alias", "--arguments" ]`. Without the duplicate `ssh-copy-id`, this would call `bin/sh.exe` with no arguments which isn't the desired behavior.